components/xsd-fu/templates-java/OMEXMLModelObject.template

```diff
@@ -129,6 +129,7 @@ public class ${klass.name} extends ${klass.parentName}
 	public ${klass.name}(Element element, OMEModel model)
 	    throws EnumerationException
 	{
+                // comment
```

Comment:
Was this addition meant to be included in this PR - see https://ci.openmicroscopy.org/view/Failing/job/BIOFORMATS-5.1-merge-generated-sources/lastSuccessfulBuild/artifact/ome-xml-model-java.diff ?